test/gtest/ucp/test_ucp_sockaddr.cc

```diff
@@ -221,13 +241,37 @@ class test_ucp_sockaddr : public ucp_test {
             ucp_request_free(send_req);
         }
 
+        if (send_status == UCS_ERR_REJECTED) {
+            return;
```

Comment:
why is it ok?

Reply:
extended if statement to be clear when it's ok